### PR TITLE
In d aadhaar number masking

### PR DESCRIPTION
**Pull request by gowthamd3**
---
When submitting a connector, please check the following conditions for your PR.

- [ ] I attest that the connector works and I verified by deploying and testing all the operations.
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (sriyen-msft) [inline apiProperties.json:1]: @gowthamd3 Please delete this file as it is added at root level.

File: certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Key",
+          "description": "The Key for this api",
+          "tooltip": "Provide your Key",
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json:9]: @gowthamd3 This should be as per the artifacts uploaded in the cert. portal.

`"description": "The API Key for this api",`
`"displayName": "API Key",`
`"tooltip": "Provide your API Key"`

File: certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#dcfffe",
+    "capabilities": [],
+    "publisher": "IN-D by Intain",
+    "stackowner": "IN-D by Intain"
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json:21]: @gowthamd3 This should be `stackOwner`, not `stackowner`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json (hunk 3 above)
Comment (gowthamd3) [reply certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json]: I have done the changes you have mentioned, pls look and let me know if any changes required.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**